src/datachain/cli/parser/pipeline.py

```diff
@@ -0,0 +1,79 @@
+from datachain.cli.parser.utils import CustomHelpFormatter
+
+
+def add_pipeline_parser(subparsers, parent_parser) -> None:
+    pipeline_helper = "Manage pipelines in Studio"
+    pipeline_description = "Commands to manage pipelines in Studio."
+    pipeline_parser = subparsers.add_parser(
+        "pipeline",
+        parents=[parent_parser],
+        description=pipeline_description,
+        help=pipeline_helper,
+        formatter_class=CustomHelpFormatter,
+    )
+    pipeline_subparser = pipeline_parser.add_subparsers(
+        dest="cmd",
+        help="Use `datachain pipeline CMD --help` to display command-specific help",
+    )
+
+    pipeline_create_help = "Create an update for dataset dependency in Studio"
+    pipeline_create_description = "Create an pipeline for a dataset dependency.\n"
```

Comment:
Create a pipeline

Please again, review carefully docs with and w/o AI. Let me know when it is done